docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -586,6 +586,21 @@ the regular expression `/^dev-[a-z0-9-]+$/`, run the following command:
 $ tsh request create --resources /teleport.example.com/namespace/mycluster/^dev-[a-z0-9-]+$
 ```
 
+Using cluster-scoped resource requests requires the role under `search_as_roles` to have the correct 
```

Comment:
I'm not sure I would expect people know what a _cluster-scoped resource_ is..

Reply:
I was following what we used for the title and the initial sentence of the section to keep it in line, should be look at renaming the section as a whole?